definitions/npm/pell_v1.x.x/flow_v0.69.x-/pell_v1.x.x.js

```diff
@@ -0,0 +1,103 @@
+declare module 'pell' {
+  declare type pell$defaultActions =
```

Comment:
You are scoped inside the module declaration, no need to prefix with `peel$`, `DefaultActions` will be fine.